### ISSUE TITLE
Socket.IO-Client-Swift Build Error : Type of expression is ambiguous without more context

### ISSUE THREAD

**Issue by zxl777** [willpowell8/LocalizationKit_iOS#27]
Swift4 ,XCODE 9

**Comment by zxl777** [comment]
Fixed it.  insert the lines bottom into podfile

```
post_install do |installer|
    installer.pods_project.targets.each do |target|
        if target.name == 'LocalizationKit' || target.name == 'Socket.IO-Client-Swift'
            print "Changing #{target.name} swift version to 3.2\n"
            target.build_configurations.each do |config|
                config.build_settings['SWIFT_VERSION'] = '3.2'
            end
        end
    end
end
```

**Comment by willpowell8** [comment]
@zxl777  thank you for this. I will add it. I am in the process of upgrading to Swift 4.

**Comment by willpowell8** [comment]
I have fixed the version of 3.0.12 as it wasnt setting the swift version to 3.2

Also released Swift 4 version so should fix the issue
